src/middleware/errorHandling/errorHandling.js

```diff
@@ -1,3 +1,13 @@
-export const consoleErrorhandler = err => {
-    console.error(err);
+import { StatusCodes } from 'http-status-codes';
+import logger from '../../logging';
+
+export const errorHandler = (err, req, res, next) => {
+    logger.error('Unhandled Error caught', {
+        errorMessage: err.message,
+        customErrorMessage: err.customErrorMessage,
+        url: req.url,
+        params: req.params
+    });
+    res.status(StatusCodes.INTERNAL_SERVER_ERROR).json({ message: err.customErrorMessage });
+    next();
```

Comment:
What can be next after sending a response?

Reply:
Thank you. Removed calling next middleware.